.gitignore

```diff
@@ -15,3 +15,4 @@ detox/*.log
 **/results
 mendixProject
 **/screenshot
+**/testProject/**
```

Comment:
are you sure? It wasn't necessary before

Reply:
Should not be necessary indeed. Although having something having something like below could be nice. But that's out of the scope of this PR.
```
*/testProject/**
!**/testProject/*.mpr
```